src/test/java/bio/terra/service/load/flight/LoadLockStepTest.java

```diff
@@ -72,11 +79,21 @@ void doStep_retryLoadLockedException() throws InterruptedException {
   }
 
   @Test
-  void doStep_throwsUnhandledException() throws InterruptedException {
-    doThrow(ConflictException.class).when(loadService).lockLoad(LOAD_TAG, FLIGHT_ID);
+  void doStep_throwsUnhandledException() {
+    doThrow(ConflictException.class).when(loadService).lockLoad(LOAD_TAG, FLIGHT_ID, DATASET_ID);
 
     assertThrows(ConflictException.class, () -> step.doStep(flightContext));
 
     assertThat(workingMap.get(LoadMapKeys.LOAD_ID, UUID.class), equalTo(null));
   }
+
+  @Test
+  void undoStep() {
+    StepResult undoResult = step.undoStep(flightContext);
+
+    assertThat(undoResult.getStepStatus(), equalTo(StepStatus.STEP_RESULT_SUCCESS));
+    assertThat(undoResult.getException().isPresent(), equalTo(false));
```

Comment:
What is the purpose of checking that the exception is missing? Is there a code path where exception is set and the flight status is `SUCCESS`?

Also, since `StepResult.getStepResultSuccess()` returns a static object, it can be compared directly:
```suggestion
    assertThat(undoResult, equalTo(StepResult.getStepResultSuccess()));
```

Reply:
It's technically possible to set an exception for a successful result, but realistically it's unlikely to happen.  Your suggested change is a good one, it sidesteps this entirely.